classes/endpoint.php

```diff
@@ -51,7 +53,7 @@ public function get_oembed_response( WP_REST_Request $request ) {
 			return new WP_Error( 'rest_oembed_invalid_url', __( 'Invalid URL.', 'oembed-api' ), array( 'status' => 404 ) );
 		}
 
-		if ( 'json' !== $request['format'] ) {
+		if ( ! in_array( $request['format'], array( 'json', 'xml' ) ) ) {
 			return new WP_Error( 'rest_oembed_invalid_format', __( 'Invalid format.', 'oembed-api' ), array( 'status' => 501 ) );
 		}
```

Comment:
(You could also do this validation via the [argument registration](http://v2.wp-api.org/extending/adding/#arguments), although that's unrelated to this PR.)